specification_templates/postprocessing.yaml

```diff
@@ -1,16 +1,16 @@
 data:
-  forecast_version: '2021_01_13.05'
+  forecast_version: '2021_01_18.01'
   scenarios:
     - 'worse'
     - 'reference'
-    - 'no_vaccine'
-    - 'vaccine_fast'
-    - 'vaccine_fast_elderly'
-    - 'best_masks'
+#    - 'no_vaccine'
+#    - 'vaccine_fast'
+#    - 'vaccine_fast_elderly'
+#    - 'best_masks'
```

Comment:
should these be deleted?

Reply:
You should ignore churn in the specifications.  I'll set up a gitignore for the ones I use for testing soon.